pkg/apiserver/filters/requestinfo.go

```diff
@@ -75,8 +75,9 @@ func WithRequestInfo(handler http.Handler, resolver request.RequestInfoResolver)
 }
 
 func injectToken(req *http.Request) (requestWithTokenContext *http.Request) {
-	authorization := req.Header.Get("Authorization")
+	authorization := req.Header.Get("X-Authorization")
```

Comment:
This kind of break change should be mentioned in README.

Reply:
Of course

Reply:
I put this into a good-first-issue. https://github.com/kubesphere-sigs/ks-devops/issues/68